test/container/1.6.0/Dockerfile.dlc.gpu

```diff
@@ -0,0 +1,28 @@
+ARG region
+from 763104351884.dkr.ecr.$region.amazonaws.com/pytorch-training:1.6.0-gpu-py36-cu110-ubuntu16.04
+
+# TODO: Remove once the 1.6.0-gpu-py3 DLC image installs mpi4py
+RUN pip3 install mpi4py==3.0.3
+
+# TODO: Remove once the 1.6.0-gpu-py3 DLC image fixes OpenSSH config
+# Configure OpenSSH so that nodes can communicate with each other
+RUN mkdir -p /var/run/sshd && \
+ sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+
+RUN rm -rf /root/.ssh/ && \
+ mkdir -p /root/.ssh/ && \
+ ssh-keygen -q -t rsa -N '' -f /root/.ssh/id_rsa && \
+ cp /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys \
+ && printf "Host *\n StrictHostKeyChecking no\n" >> /root/.ssh/config
+
+# TODO: Remove once the 1.6.0-gpu-py3 DLC image fixes MPI config
+# Comment line in MPI config to prevent mutually exclusive MCA settings
+RUN sed -i '62,62 s/^/#/' /home/.openmpi/etc/openmpi-mca-params.conf
```

Comment:
https://github.com/aws/PRIVATE-deep-learning-containers/blob/4d4307b368dc10708556e73face8cd4efa2da4dd/pytorch/training/docker/1.6.0/py3/cu101/Dockerfile.gpu#L177
This was the line which was to be removed.

This was applicable for cu101.

However, reinvent DLC which was cu110 doesn't have that particular exclude line
https://github.com/aws/PRIVATE-deep-learning-containers/blob/private-master/pytorch/training/docker/1.6.0/py3/cu110/Dockerfile.gpu#L270-L278

Hence we don't need to run this commenting RUN statement anymore for cu110

Reply:
It also looks like all of the other TODOs can be removed from the Dockerfile. Is that correct?

mpi4py: https://github.com/aws/PRIVATE-deep-learning-containers/blob/4d4307b368dc10708556e73face8cd4efa2da4dd/pytorch/training/docker/1.6.0/py3/cu101/Dockerfile.gpu#L152
OpenSSH: https://github.com/aws/PRIVATE-deep-learning-containers/blob/4d4307b368dc10708556e73face8cd4efa2da4dd/pytorch/training/docker/1.6.0/py3/cu101/Dockerfile.gpu#L181

Reply:
Yes please!